job-service-db/src/main/resources/procedures/deleteDependentJobs.sql

```diff
@@ -0,0 +1,35 @@
+--
+-- Copyright 2016-2020 Micro Focus or one of its affiliates.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+ *  Name: delete_dependent_jobs
```

Comment:
Also rename the file in line with this change.

```suggestion
 *  Name: delete_dependent_job
```